### PR TITLE
fix: language and grammar in the root README

### PR DESCRIPTION
**Pull request by nfreear**
# Pull Request Template

## PR Checklist

- [x] I have run `npm test` locally and all tests are passing. __N/A__
- [x] I have added/updated tests for any new behavior. __N/A__
- [x] If this is a significant change, an issue has already been created where the problem / solution was discussed: [__N/A__, or add link to issue here]

## PR Description

I noticed some grammar and language issues, so thought I'd try to make the document clearer.

I hope this helps.

Yours,

Nick

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,33 +16,33 @@
 "NLP.js" is a general natural language utility for nodejs. Currently supporting:
 
 - Guess the language of a phrase
-- Fast levenshtein distance of two strings
-- Search the best substring of a string with less levenshtein distance to a given pattern.
+- Fast _Levenshtein_ distance of two strings
+- Search the best substring of a string with less _Levenshtein_ distance to a given pattern.
 - Get stemmers and tokenizers for several languages.
 - Sentiment Analysis for phrases (with negation support).
 - Named Entity Recognition and management, multilanguage, and accepting similar strings, so the introduced text does not need to be exact.
```
Comment (jesus-seijas-sp) [inline README.md:23]: This line can be removed, as it's replaced by the next one.

**Review by jesus-seijas-sp — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (nfreear) [reply README.md]: Hi @jesus-seijas-sp,

Sorry, do you mean that the "Fast levenshstein ..." line duplicates the "Search the best substring ..." line?

Btw, I realised I introduced a duplicated line, so removed that ... ;)

I have a few fixes for the Table of contents links, then I'll be ready.

Ta,

Nick

**Comment by nfreear** [comment]
Note, there are two options for the _License_ link in the table of contents:

```markdown
- [License](#license)
```

Or

```markdown
- [License](./LICENSE.md)
```

I went with the first option, but it's your call.

N;)

File: README.md (hunk 1 above)
Comment (jesus-seijas-sp) [reply README.md]: Hello, the line was "Named Entity Recognition...". It's already fixed.

**Comment by jesus-seijas-sp** [comment]
> Note, there are two options for the _License_ link in the table of contents:
> 
> ```gfm
> - [License](#license)
> ```
> 
> Or
> 
> ```gfm
> - [License](./LICENSE.md)
> ```
> 
> I went with the first option, but it's your call.
> 
> Also, at the risk of sounding patronising, thank you for the growing documentation _in English!_ The changes I'm suggesting are small in the big scheme of things ...
> 
> N;)

Yes, first option is preferred

**Comment by jesus-seijas-sp** [comment]
Hello @nfreear , when this PR is ready to merge let me know (currently is marked as draft).
Also, I added the label "hacktoberfest-accepted", so it's elegible for hacktoberfest.

**Review by jesus-seijas-sp — APPROVED** [review]

**Comment by nfreear** [comment]
Hi @jesus-seijas-sp,

... Woops, forgot it was marked as '_draft_'! ... all ready to merge by the looks of things. Best wishes, 

Nick

**Pull request merged**